frontend/pkg/frontend/middleware_tracing.go

```diff
@@ -0,0 +1,28 @@
+package frontend
+
+// Copyright (c) Microsoft Corporation.
+// Licensed under the Apache License 2.0.
+
+import (
+	"fmt"
+	"net/http"
+	"strings"
+
+	"go.opentelemetry.io/contrib/instrumentation/net/http/otelhttp"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/trace"
+)
+
+func MiddlewareTracing(w http.ResponseWriter, r *http.Request, next http.HandlerFunc) {
```

Comment:
(suggestion) can we add a comment explaining the purpose of the middleware for future readers?